perun-base/src/main/resources/perun-roles.yml

```diff
@@ -5204,22 +5233,33 @@ perun_policies:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: User
```

Comment:
I feel like whenever working with UES, we should check both the UES and User associated (so in `updateUserExtSource_UserExtSource_policy` as well) @Johaney-s  @cuadradek thoughts?

Reply:
here I agree; when deleting user I'm not so sure, but better make it stricter now and if needed we'll remove it later... so okay by me